tests/bdd/features/csi/controller/test_controller.py

```diff
@@ -999,6 +1031,12 @@ def check_unpublish_not_existing_volume(unpublish_not_existing_volume):
     ), "Volume unpuplishing succeeded with unexpected response"
 
 
+@then("volume should remain as published")
+def volume_remains_published():
+    """volume should remain as published."""
+    pass
```

Comment:
need to check it's published here?

Reply:
The previous step is checking that the volume target isn't deleted and remains available on the original node , so simply `pass`ing here.